rexray/cli/cmds_volume.go

```diff
@@ -66,14 +66,13 @@ func (c *CLI) initVolumeCmds() {
 			}
 			if c.volumeID != "" || c.volumeName != "" {
 				for _, v := range vols {
-					if strings.ToLower(v.ID) == strings.ToLower(c.volumeID) ||
-						strings.ToLower(v.Name) == strings.ToLower(c.volumeName) {
+					if (c.volumeID != "" && strings.ToLower(v.ID) == strings.ToLower(c.volumeID)) ||
```

Comment:
Hi @proudh,

This looks good, thank you! Could you please do me a favor and update the logic? I recently read [an article](https://medium.com/@benbjohnson/go-walkthrough-bytes-strings-packages-499be9f4b5bd) by @benbjohnson on ways to optimize Go code, and string comparisons are one area that can be greatly improved. To achieve efficient, case-insensitive string comparison, please use [`string.EqualFold`](https://golang.org/pkg/strings/#EqualFold) instead. Memory is allocated with every `strings.ToLower` call, and this is avoided by using the `EqualFold` comparison function. I've started to use it wherever I would normally have used a `ToLower` or `ToUpper` to perform case-insensitive comparison. 

Thanks!